docs/our-approach-to-data.md

```diff
@@ -332,9 +332,8 @@ export const items = createReducer( defaultState, {
 	[DESERIALIZE]: state => fromJS( state )
 } );
 ```
-If your reducer state is already a plain object, you may choose to omit `SERIALIZE` and `DESERIALIZE` handlers, or 
-simply define them as returning the current state. However, please note that the subtree can still see errors from 
-changing data shapes, as described below.
+If your reducer state is already a plain object, the `SERIALIZE` and `DESERIALIZE` handlers are not needed.
+However, please note that the subtree can still see errors from changing data shapes, as described below.
```

Comment:
This may have already existed in ambiguity, but what happens if we have simple data for our reducer shape?